SummerProject/config/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-from local_settings import *
```

Comment:
Why this is removed?

Reply:
Because, when I did hard reset, local_settings just disappeared from my file, but seems to previous comment, I'm gonna fix it.
Probably, add again local_settings with my DB user, right?